### PR TITLE
Address stylecheck warnings

### PR DESCRIPTION
**Pull request by antoineco**
Addresses the following style-related linter warnings. Mostly:

- double imports
- common abbreviations not capitalized
- method receivers with inconsistent variable names
- capitalized error strings

```
pkg/sources/adapter/ocimetricssource/config.go:33:2: ST1003: struct field OracleApiKey should be OracleAPIKey (stylecheck)
        OracleApiKey            string `envconfig:"ORACLE_API_PRIVATE_KEY" required:"true"`
        ^
pkg/sources/adapter/ocimetricssource/config.go:34:2: ST1003: struct field OracleApiKeyPassphrase should be OracleAPIKeyPassphrase (stylecheck)
        OracleApiKeyPassphrase  string `envconfig:"ORACLE_API_PRIVATE_KEY_PASSPHRASE" required:"true"`
        ^
pkg/sources/adapter/ocimetricssource/config.go:35:2: ST1003: struct field OracleApiKeyFingerprint should be OracleAPIKeyFingerprint (stylecheck)
        OracleApiKeyFingerprint string `envconfig:"ORACLE_API_PRIVATE_KEY_FINGERPRINT" required:"true"`
        ^
pkg/sources/adapter/twiliosource/types.go:29:2: ST1003: struct field ApiVersion should be APIVersion (stylecheck)
        ApiVersion    string `json:"api_version"`
        ^
pkg/sources/aws/iam/iam.go:29:2: ST1003: struct field Id should be ID (stylecheck)
        Id        string            `json:"Id,omitempty"`
        ^
pkg/sources/aws/sqs/sqs.go:52:2: ST1003: var polJson should be polJSON (stylecheck)
        polJson, err := json.Marshal(pol)
        ^
pkg/targets/adapter/googlecloudworkflowstarget/types.go:21:2: ST1003: struct field WorkflowId should be WorkflowID (stylecheck)
        WorkflowId string `json:"workflow_id"`
        ^
pkg/targets/adapter/googlecloudworkflowstarget/types.go:25:3: ST1003: struct field RevisionId should be RevisionID (stylecheck)
                RevisionId  string             `json:"revision_id"`
                ^
pkg/targets/adapter/hasuratarget/adapter.go:37:7: ST1003: const baseUrl should be baseURL (stylecheck)
const baseUrl = "/v1/graphql"
      ^
pkg/targets/adapter/oracletarget/config.go:31:2: ST1003: struct field OracleApiKey should be OracleAPIKey (stylecheck)
        OracleApiKey            string `envconfig:"ORACLE_API_PRIVATE_KEY" required:"true"`
        ^
pkg/targets/adapter/oracletarget/config.go:32:2: ST1003: struct field OracleApiKeyPassphrase should be OracleAPIKeyPassphrase (stylecheck)
        OracleApiKeyPassphrase  string `envconfig:"ORACLE_API_PRIVATE_KEY_PASSPHRASE" required:"true"`
        ^
pkg/targets/adapter/oracletarget/config.go:33:2: ST1003: struct field OracleApiKeyFingerprint should be OracleAPIKeyFingerprint (stylecheck)
        OracleApiKeyFingerprint string `envconfig:"ORACLE_API_PRIVATE_KEY_FINGERPRINT" required:"true"`
        ^
pkg/apis/sources/v1alpha1/googlecloudbilling_types.go:50:2: ST1003: struct field BillingAccountId should be BillingAccountID (stylecheck)
        BillingAccountId string `json:"billingAccountId"`
        ^
pkg/apis/sources/v1alpha1/googlecloudbilling_types.go:55:2: ST1003: struct field BudgetId should be BudgetID (stylecheck)
        BudgetId string `json:"budgetId"`
        ^
pkg/apis/sources/v1alpha1/ocimetricssource_types.go:58:2: ST1003: struct field OracleApiPrivateKey should be OracleAPIPrivateKey (stylecheck)
        OracleApiPrivateKey ValueFromField `json:"oracleApiPrivateKey"`
        ^
pkg/apis/sources/v1alpha1/ocimetricssource_types.go:61:2: ST1003: struct field OracleApiPrivateKeyPassphrase should be OracleAPIPrivateKeyPassphrase (stylecheck)
        OracleApiPrivateKeyPassphrase ValueFromField `json:"oracleApiPrivateKeyPassphrase"`
        ^
pkg/apis/sources/v1alpha1/ocimetricssource_types.go:64:2: ST1003: struct field OracleApiPrivateKeyFingerprint should be OracleAPIPrivateKeyFingerprint (stylecheck)
        OracleApiPrivateKeyFingerprint ValueFromField `json:"oracleApiPrivateKeyFingerprint"`
        ^
pkg/apis/targets/v1alpha1/oracle_types.go:53:2: ST1003: struct field OracleApiPrivateKey should be OracleAPIPrivateKey (stylecheck)
        OracleApiPrivateKey SecretValueFromSource `json:"oracleApiPrivateKey"`
        ^
pkg/apis/targets/v1alpha1/oracle_types.go:56:2: ST1003: struct field OracleApiPrivateKeyPassphrase should be OracleAPIPrivateKeyPassphrase (stylecheck)
        OracleApiPrivateKeyPassphrase SecretValueFromSource `json:"oracleApiPrivateKeyPassphrase"`
        ^
pkg/apis/targets/v1alpha1/oracle_types.go:59:2: ST1003: struct field OracleApiPrivateKeyFingerprint should be OracleAPIPrivateKeyFingerprint (stylecheck)
        OracleApiPrivateKeyFingerprint SecretValueFromSource `json:"oracleApiPrivateKeyFingerprint"`
        ^
pkg/apis/targets/v1alpha1/uipath_lifecycle.go:31:2: ST1003: const EventTypeUiPathStartJob should be EventTypeUIPathStartJob (stylecheck)
        EventTypeUiPathStartJob = "io.triggermesh.uipath.job.start"
        ^
pkg/apis/targets/v1alpha1/uipath_lifecycle.go:33:2: ST1003: const EventTypeUiPathQueuePost should be EventTypeUIPathQueuePost (stylecheck)
        EventTypeUiPathQueuePost = "io.triggermesh.uipath.queue.post"
        ^
pkg/apis/targets/v1alpha1/uipath_lifecycle.go:37:5: ST1003: var UiPathCondSet should be UIPathCondSet (stylecheck)
var UiPathCondSet = apis.NewLivingConditionSet(
    ^
pkg/apis/targets/v1alpha1/uipath_types.go:34:6: ST1003: type UiPathTarget should be UIPathTarget (stylecheck)
type UiPathTarget struct {
     ^
pkg/apis/targets/v1alpha1/uipath_types.go:52:6: ST1003: type UiPathTargetSpec should be UIPathTargetSpec (stylecheck)
type UiPathTargetSpec struct {
     ^
pkg/apis/targets/v1alpha1/uipath_types.go:70:6: ST1003: type UiPathTargetStatus should be UIPathTargetStatus (stylecheck)
type UiPathTargetStatus struct {
     ^
pkg/apis/targets/v1alpha1/uipath_types.go:81:6: ST1003: type UiPathTargetList should be UIPathTargetList (stylecheck)
type UiPathTargetList struct {
     ^
pkg/sources/adapter/awscloudwatchsource/adapter_test.go:94:3: ST1003: const metricId should be metricID (stylecheck)
                metricId         = "testmetrics"
                ^
pkg/sources/adapter/awscloudwatchsource/adapter_test.go:190:3: ST1003: const metricId should be metricID (stylecheck)
                metricId         = "testmetrics"
                ^
pkg/sources/adapter/slacksource/slack.go:34:7: ST1003: const apiAppIdCeExtension should be apiAppIDCeExtension (stylecheck)
const apiAppIdCeExtension = "comslackapiappid"
      ^
pkg/sources/reconciler/googlecloudbillingsource/billing.go:165:6: ST1003: func generateBudgetId should be generateBudgetID (stylecheck)
func generateBudgetId(s *v1alpha1.GoogleCloudBillingSource) string {
     ^
pkg/sources/reconciler/ocimetricssource/adapter.go:37:2: ST1003: const oracleApiKey should be oracleAPIKey (stylecheck)
        oracleApiKey            = "ORACLE_API_PRIVATE_KEY"
        ^
pkg/sources/reconciler/ocimetricssource/adapter.go:38:2: ST1003: const oracleApiKeyPassphrase should be oracleAPIKeyPassphrase (stylecheck)
        oracleApiKeyPassphrase  = "ORACLE_API_PRIVATE_KEY_PASSPHRASE"
        ^
pkg/sources/reconciler/ocimetricssource/adapter.go:39:2: ST1003: const oracleApiKeyFingerprint should be oracleAPIKeyFingerprint (stylecheck)
        oracleApiKeyFingerprint = "ORACLE_API_PRIVATE_KEY_FINGERPRINT"
        ^
pkg/targets/adapter/googlesheettarget/adapter_test.go:87:4: ST1003: var mockUrl should be mockURL (stylecheck)
                        mockUrl := fmt.Sprintf("/v4/spreadsheets/%s/values/!2:%d", sheetID, maxSheetRow+1)
                        ^
pkg/targets/adapter/googlesheettarget/adapter_test.go:188:4: ST1003: var mockUrl should be mockURL (stylecheck)
                        mockUrl := fmt.Sprintf("/v4/spreadsheets/%s", sheetID)
                        ^
pkg/targets/adapter/googlesheettarget/adapter_test.go:253:4: ST1003: var mockUrl should be mockURL (stylecheck)
                        mockUrl := fmt.Sprintf("/v4/spreadsheets/%s:batchUpdate", sheetID)
                        ^
pkg/targets/reconciler/oracletarget/target_adapter.go:68:6: ST1003: var fnId should be fnID (stylecheck)
        var fnId string
            ^
pkg/apis/flow/v1alpha1/transformation_lifecycle.go:34:26: ST1016: methods on the same type should have the same receiver name (seen 1x "as", 4x "t") (stylecheck)
func (t *Transformation) GetConditionSet() apis.ConditionSet {
                         ^
pkg/apis/targets/v1alpha1/tekton_lifecycle.go:59:22: ST1016: methods on the same type should have the same receiver name (seen 1x "t", 3x "s") (stylecheck)
func (*TektonTarget) AcceptedEventTypes() []string {
                     ^
pkg/apis/targets/v1alpha1/oracle_lifecycle.go:84:24: ST1016: methods on the same type should have the same receiver name (seen 1x "o", 2x "s") (stylecheck)
func (s *OracleTarget) GetConditionSet() apis.ConditionSet {
                       ^
pkg/apis/targets/v1alpha1/googlesheet_lifecycle.go:36:35: ST1016: methods on the same type should have the same receiver name (seen 1x "a", 6x "s") (stylecheck)
func (s *GoogleSheetTargetStatus) GetCondition(t apis.ConditionType) *apis.Condition {
                                  ^
pkg/apis/targets/v1alpha1/slack_lifecycle.go:58:21: ST1016: methods on the same type should have the same receiver name (seen 1x "t", 3x "s") (stylecheck)
func (*SlackTarget) AcceptedEventTypes() []string {
                    ^
pkg/sources/reconciler/googlecloudauditlogssource/adapter.go:57:22: ST1016: methods on the same type should have the same receiver name (seen 1x "c", 4x "r") (stylecheck)
func (r *Reconciler) BuildAdapter(src v1alpha1.EventSource, sinkURI *apis.URL) *appsv1.Deployment {
                     ^
pkg/sources/reconciler/googlecloudrepositoriessource/adapter.go:57:22: ST1016: methods on the same type should have the same receiver name (seen 1x "c", 4x "r") (stylecheck)
func (r *Reconciler) BuildAdapter(src v1alpha1.EventSource, sinkURI *apis.URL) *appsv1.Deployment {
                     ^
pkg/routing/reconciler/filter/reconciler.go:24:2: ST1019: package "github.com/triggermesh/triggermesh/pkg/apis/routing/v1alpha1" is being imported more than once (stylecheck)
        "github.com/triggermesh/triggermesh/pkg/apis/routing/v1alpha1"
        ^
pkg/routing/reconciler/filter/reconciler.go:25:2: ST1019(related information): other import of "github.com/triggermesh/triggermesh/pkg/apis/routing/v1alpha1" (stylecheck)
        routingv1alpha1 "github.com/triggermesh/triggermesh/pkg/apis/routing/v1alpha1"
        ^
pkg/routing/reconciler/splitter/reconciler.go:24:2: ST1019: package "github.com/triggermesh/triggermesh/pkg/apis/routing/v1alpha1" is being imported more than once (stylecheck)
        "github.com/triggermesh/triggermesh/pkg/apis/routing/v1alpha1"
        ^
pkg/routing/reconciler/splitter/reconciler.go:25:2: ST1019(related information): other import of "github.com/triggermesh/triggermesh/pkg/apis/routing/v1alpha1" (stylecheck)
        routingv1alpha1 "github.com/triggermesh/triggermesh/pkg/apis/routing/v1alpha1"
        ^
pkg/targets/reconciler/alibabaosstarget/adapter.go:26:2: ST1019: package "github.com/triggermesh/triggermesh/pkg/apis/targets/v1alpha1" is being imported more than once (stylecheck)
        "github.com/triggermesh/triggermesh/pkg/apis/targets/v1alpha1"
        ^
```

### CODE REVIEW

File: pkg/apis/targets/v1alpha1/googlesheet_lifecycle.go [hunk 1]
```diff
@@ -103,8 +103,8 @@ func (s *GoogleSheetTargetStatus) PropagateAvailability(ksvc *servingv1.Service)
 }
 
 // MarkNoKService sets the condition that the service is not ready
-func (a *GoogleSheetTargetStatus) MarkNoService(reason, messageFormat string, messageA ...interface{}) {
-	AwsCondSet.Manage(a).MarkFalse(ConditionServiceReady, reason, messageFormat, messageA...)
+func (s *GoogleSheetTargetStatus) MarkNoService(reason, messageFormat string, messageA ...interface{}) {
+	AwsCondSet.Manage(s).MarkFalse(ConditionServiceReady, reason, messageFormat, messageA...)
```
Comment (JeffNeff) [inline pkg/apis/targets/v1alpha1/googlesheet_lifecycle.go:107]: oh man.. lol just noticed this xd

Comment (JeffNeff) [reply pkg/apis/targets/v1alpha1/googlesheet_lifecycle.go:107]: https://github.com/triggermesh/triggermesh/issues/74

Comment (antoineco) [reply pkg/apis/targets/v1alpha1/googlesheet_lifecycle.go:107]: Uh, oh. AWS in a Google target 😄

File: pkg/targets/adapter/slacktarget/slack/webapi.go [hunk 2]
```diff
@@ -49,16 +49,16 @@ type webAPIClient struct {
 func (c *webAPIClient) Do(methodURL string, data []byte) (Response, error) {
 	method, ok := c.methods[methodURL]
 	if !ok {
-		return nil, fmt.Errorf("Slack method %q not supported", methodURL)
+		return nil, fmt.Errorf("Slack method %q not supported", methodURL) //nolint:stylecheck
```
Comment (JeffNeff) [inline pkg/targets/adapter/slacktarget/slack/webapi.go:52]: Why did you change the capitalization of `Error` in the above and mark these as `nolint`?  just wondering

File: pkg/targets/reconciler/awscomprehendtarget/adapter.go [hunk 3]
```diff
@@ -24,7 +24,6 @@ import (
 	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
 
 	"github.com/triggermesh/triggermesh/pkg/apis/targets/v1alpha1"
-	awscomprehendv1alpha1 "github.com/triggermesh/triggermesh/pkg/apis/targets/v1alpha1"
```
Comment (JeffNeff) [inline pkg/targets/reconciler/awscomprehendtarget/adapter.go:27]: Im glad to see these go. I never understood why we did this.

**Review by JeffNeff — APPROVED** [review]

File: pkg/targets/reconciler/awscomprehendtarget/adapter.go (hunk 3 above)
Comment (antoineco) [reply pkg/targets/reconciler/awscomprehendtarget/adapter.go:27]: It's a leftover from the times when each target had its own controller, `pkg`, etc. I guess.

File: pkg/targets/adapter/slacktarget/slack/webapi.go (hunk 2 above)
Comment (antoineco) [reply pkg/targets/adapter/slacktarget/slack/webapi.go:52]: Because Slack is a brand name so I didn't want to change it to `slack`. That would have made the linter happy too though.
